Catering WebAPI/Entities/Customer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Catering_WebAPI.Entities
+{
+    public class Customer
+    {
+        public int CustomerId { get; set; }
+        public string Name { get; set; } = string.Empty;
```

Comment:
Name, Surname shouldnt be empy. Use Requier and MinLength Attributes. I would recommend using MaxLengh attribute as well